### PR TITLE
Changed status response and added 5 node setup to Travis

### LINKED ISSUE #208: update status message format
Upon request, load balancer returns a string version of status message. We need to change the return message format to JSON object. The suggested format is as follows.

```json
{
    "components" :
        [{
            "role" : "<load_balancer> | <execution_node>",
            "host" : "<host_name> | <ip_address>",
            "port" : "port_number",
            "status" : "<up> | <down>"
    }],
    "job_queue_length" : "<number>",
    "time" : "<time-stamp>"
}
```

### PR DESCRIPTION
**Pull request by AnkshitJain**
Closes #208 and updates the 5 execution node structure for testing on Travis

### CODE REVIEW

**Comment by AnkshitJain** [comment]
The 2 code climate issues deal with replication of the code. This will be dealt later in the refactoring of the js files.

File: deploy/configs/load_balancer/nodes_data_conf.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "port": "80"
   },
   "host_port": {
+    "hostname": "localhost",
```
Comment (prasadtalasila) [inline deploy/configs/load_balancer/nodes_data_conf.json]: conceptually, the placement of hostname under host_port would be inappropriate. Given that we give this field a name **load_balancer** in [conf.json](https://github.com/AutolabJS/AutolabJS/blob/master/deploy/configs/execution_nodes/conf.json#L1) of execution nodes, it might be appropriate to use the same object structure here also.

File: load_balancer/load_balancer.js [hunk 2]
```diff
@@ -25,60 +25,96 @@ app.use(express.static(__dirname + '/public'));
 app.use(bodyParser.urlencoded({extended: true}));
 app.use(bodyParser.json());
 
+function checkNodeConn(node, response) {
```
Comment (prasadtalasila) [inline load_balancer/load_balancer.js]: It would be nice to keep this in a separate file (technically a module) named status.js, import that object and use it here. Please write the status.js file in ES6 coding conventions and put a unit test around it. All of this work seems like worth a separate PR. So for now, remove this update to connection check and push the five node installation part in this PR.

In another PR, you can send in the status check updates with the suggested changes.

File: tests/extract_run_test.sh [hunk 3]
```diff
@@ -27,5 +28,5 @@ mkdir "$lab"
 cd "$lab" || exit
 
 # copy sample unit tests and execute
-cp -rf ../../../../docs/examples/unit_tests/* .
+cp -rf ../../../../../docs/examples/unit_tests/* . || exit
```
Comment (prasadtalasila) [inline tests/extract_run_test.sh]: why is this change necessary?

File: tests/functional_tests/helper_scripts/website_load/status_check.sh [hunk 4]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```
Comment (prasadtalasila) [inline tests/functional_tests/helper_scripts/website_load/status_check.sh]: As per suggestion given in one of the previous comments, please move this to a separate PR.

File: tests/nodes_data_conf_single_node.json [hunk 5]
```diff
@@ -1,25 +0,0 @@
-{
```
Comment (prasadtalasila) [inline tests/nodes_data_conf_single_node.json]: Why was this configuration file of load balancer lurking at the top-level of the **tests** directory?

**Comment by prasadtalasila** [comment]
@AnkshitJain We use array's push and pop methods to manage the stack of available execution nodes. Because of that reason, we are exercising only one execution node in the tests on travis. please see [travis log](https://travis-ci.org/AutolabJS/AutolabJS/builds/291082679#L2067). All other execution nodes remain sidelined. 

The solution would be to introduce concurrency in tests. What if we can schedule concurrent bats tests by push the invocation of bats files into background? The test objective is to have concurrent utilization of execution nodes. If there is a better approach to this problem, then I am all ears.

File: deploy/configs/load_balancer/nodes_data_conf.json (hunk 1 above)
Comment (AnkshitJain) [reply deploy/configs/load_balancer/nodes_data_conf.json]: Having a field **load_balancer** seems correct.

File: load_balancer/load_balancer.js (hunk 2 above)
Comment (AnkshitJain) [reply load_balancer/load_balancer.js]: It will be moved to the next PR and in a separate module.

File: tests/extract_run_test.sh (hunk 3 above)
Comment (AnkshitJain) [reply tests/extract_run_test.sh]: Since we have execution_node_1, ... , execution_node_5 inside execution_nodes directory as per the new structure, while the extract_run.sh still remains in the respective execution_node_1, ...  , execution_node_5 directories, this change is required. The additional || exit has been removed.

File: tests/nodes_data_conf_single_node.json (hunk 5 above)
Comment (AnkshitJain) [reply tests/nodes_data_conf_single_node.json]: When the structure was changed, Travis was still running on a single node setup. This file was meant to be replaced before the Travis testing in place of the 5 node configuration file that exists in deploy/configs/load_balancer. Now that we have shifted to five node setup on Travis as well, we do not require this configuration file.

**Comment by AnkshitJain** [comment]
The suggested concurrent test for multiple execution nodes has already been listed in #206.
